<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckTranslation</class>
 <widget class="QMainWindow" name="CheckTranslation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="feedbackTable">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>审核译文</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="srcSubText">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="accept">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>接受</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="translators">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>201</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="complete">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>合并译文</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
